services/shuttle-tower/src/lib.rs

```diff
@@ -58,7 +58,8 @@ where
     /// Takes the service that is returned by the user in their [shuttle_runtime::main] function
     /// and binds to an address passed in by shuttle.
     async fn bind(mut self, addr: SocketAddr) -> Result<(), Error> {
-        let shared = tower::make::Shared::new(self.0);
+        let shared = tower::make::Shared::new(HealthCheck::new(self.0));
```

Comment:
This is great!

Reply:
thanks :pray: btw this was my first time implementing a `tower::Service`,  I watched this [youtube stream](https://youtu.be/16sU1q8OeeI) from **David Pedersen** to learn how to do it